doc/nrf/releases/release-notes-latest.rst

```diff
@@ -20,8 +20,18 @@ See the change log for each library in the :doc:`nrfxlib documentation <nrfxlib:
 sdk-zephyr
 ==========
 
+.. NOTE TO MAINTAINERS: the latest zephyr commit appears in multiple places; make sure you update them all.
+
 The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``4ef29b34e3``, plus some |NCS| specific additions.
 
+For a complete list of |NCS| specific commits, run this command from the :file:`ncs/zephyr` repository::
```

Comment:
I suggest to switch these two commands around, because in the introduction, we mention upstream commits before NCS specific additions, and we use the same order for the lists below. So I would follow the same order here as well.